Integrations/integration-secdo.yml

```diff
@@ -0,0 +1,135 @@
+description: Secdo's automated incident response platform hunts threats in real time
+  and delivers an endpoint detection and response solution.
+configuration:
+- display: Secdo Server
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Secdo Key
+  name: key
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Do not validate server certificate (insecute)
+  name: insecure
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Company name
+  name: company
+  defaultvalue: ""
+  type: 0
+  required: true
+script:
+  script: |+
+    var sendRequest = function() {
+        var requestUrl = 'https://' + params.server + '/publicapi/v2/load_ioc/';
```

Comment:
you should add `https://` only if not exists in the server param

Reply:
Added